protobuf/echo/Echo.hpp

```diff
@@ -198,7 +199,7 @@ namespace services
         : public infra::IntrusiveList<ServiceProxy>::NodeType
     {
     public:
-        ServiceProxy(Echo& echo, uint32_t id, uint32_t maxMessageSize);
+        ServiceProxy(Echo& echo, EMIL_MAYBE_UNUSED uint32_t serviceId, uint32_t maxMessageSize);
```

Comment:
This is just the declaration; does it need an [[unused]] tag as well?

Reply:
Probably not, no. If we keep the parameter, I will remove this.